README.md

```diff
@@ -47,3 +47,34 @@ print(get_speedify_settings())
 ```
 
 For the full list of settings, build the sphinx documentation by running make html in the docs/ folder.
+
+## Changelog
+
+### Release 12.5.x
+
+Added
+  - `dns(str)`
+  - `streamtest()`
+  - `directory(str)`
+  - `show(str)`
```

Comment:
The generic `show` command is not needed here. Instead in this library we have `show_adapters`, `show_settings`, etc. Anything new added to `show` in the CLI should get a function here.

Reply:
Will do